pkg/ebpf/tracee.go

```diff
@@ -1641,11 +1641,12 @@ func computeFileHashAtPath(fileName string) (string, error) {
 }
 
 func computeFileHash(file *os.File) (string, error) {
-	h := sha256.New()
+	h := miniosha.New()
```

Comment:
Will this work cross platform? That was my initial concern with using this library, that I couldn't assess if it would work everywhere.

Reply:
I just checked their source code, they do a hardware check themselves and fallback to the default stdlib hash if needed. So we should be ok on this!

Reply:
I was doing the same. Neat!